plugins/Live/Widgets/GetVisitorProfilePopup.php

```diff
@@ -36,8 +36,4 @@ public static function configure(WidgetConfig $config)
             $config->disable();
         }
     }
-
-    public function render()
-    {
-    }
```

Comment:
The method is meant to return a string. Furthermore the method of the base class also returns nothing, so there is no need to overwrite it.